Tests/helper.py

```diff
@@ -250,25 +251,27 @@ def tostring(im: Image.Image, string_format: str, **options: Any) -> bytes:
     return out.getvalue()
 
 
-def hopper(mode: str | None = None, cache: dict[str, Image.Image] = {}) -> Image.Image:
+def hopper(mode: str | None = None) -> Image.Image:
+    # Use caching to reduce reading from disk but so an original copy is
+    # returned each time and the cached image isn't modified by tests
+    # (for fast, isolated, repeatable tests).
```

Comment:
This seems to be a run-on sentence, and some of the words seem to be misplaced. Perhaps this instead:

> Use caching to reduce reading from disk, but return a copy so that the cached image isn't modified by the tests.

Reply:
I think the original does make sense, but it can definitely be improved, and your suggestion is much clearer 👍 

Will update.